tests/checkpointing/test_model_checkpoint.py

```diff
@@ -512,3 +513,61 @@ def validation_epoch_end(self, outputs):
 
     # check that last one is also the best one
     assert trainer.dev_debugger.checkpoint_callback_history[-1]['epoch'] == len(monitor) - 1
+
+
+def test_model_torch_save(tmpdir):
```

Comment:
let's move all these to a special file as it is not about checkpointing and you also save the trainer
maybe tests/test_saving..py ?